debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/DateTimeFormat.java

```diff
@@ -98,17 +122,17 @@ public static DateTimeFormat get() {
 
         @Override
         public long timestamp(final String s) {
-                return format(TS_FORMAT_PATTERN, s, () -> NanoTimestamp.toEpochNanos(LocalDateTime.parse(s, TS_FORMAT), null));
+                return format(TS_FORMAT_PATTERN_HINT, s, () -> NanoTimestamp.toEpochNanos(LocalDateTime.parse(s, TS_FORMAT), null));
```

Comment:
I don't quite remember all the details, but one part of the problem was the use of `toEpochNanos()`, which still is in use here. So I'm wondering whether there are still problems potentially. Could you check with the comments on [DBZ-1205](https://issues.jboss.org/browse/DBZ-1205)?

Reply:
I'll take a look at DBZ-1205. Although it is probably better to fix it in a separate PR.

Reply:
@gunnarmorling OK, I was able to avoid using toEpochNanos during conversion and converted timestamps to OffsetDateTime directly so it should address the DBZ-1205. But there are a lot more changes compared to this PR so I'm going to create followup PR.

Reply:
@gunnarmorling I've squashed commits into one

Reply:
My email is `ivanobulo@gmail.com`

Reply:
OK, I'll add DBZ-1205 changes to this PR.